src/components/Entry/DomainArchitectures/index.js

```diff
@@ -167,37 +175,39 @@ class DomainArchitectures extends PureComponent {
                       protein: { db: 'UniProt' },
                       entry: { db: 'InterPro', accession: mainAccession },
                     },
-                    search: { ida: obj.IDA_FK },
+                    search: { ida: obj.ida_id },
                   }}
                 >
                   There {obj.unique_proteins > 1 ? 'are' : 'is'}{' '}
                   {obj.unique_proteins} proteins{' '}
                 </Link>
                 with this architecture:<br />
-                {idaObj.domains.map(d => (
-                  <span key={d.accessions.join('|')}>
-                    {d.accessions.map(acc => (
-                      <Link
-                        key={acc}
-                        to={{
-                          description: {
-                            main: { key: 'entry' },
-                            entry: { db: 'InterPro', accession: acc },
-                          },
-                        }}
-                      >
-                        {' '}
-                        {acc}{' '}
-                      </Link>
-                    ))}{' '}
-                    -
+                {idaObj.accessions.map(d => (
+                  <span key={d}>
+                    <Link
+                      to={{
+                        description: {
+                          main: { key: 'entry' },
+                          entry: { db: 'InterPro', accession: d },
+                        },
+                      }}
+                    >
+                      {' '}
+                      {d}{' '}
+                    </Link>{' '}
+                    -{' '}
                   </span>
                 ))}
-                <IDAProtVista matches={idaObj.domains} length={idaObj.length} />
+                <IDAProtVista matches={idaObj.domains} length={1000} />
```

Comment:
Why was that changed to a fixed value, is that temporary?

Reply:
Yes, it is now constant because the domains don't have length, so everything is now calculated on an hypothetical length of 1000, however I should've  use the previously defined constant length. (line 61)

Reply:
Ah, OK, makes sense. Do you want to do the change to use a common const value?

Reply:
Done!